woodwork/statistics_utils.py

```diff
@@ -204,8 +212,17 @@ def _get_mutual_information_dict(dataframe, num_bins=10, nrows=None, include_ind
     if set(not_null_cols) != set(valid_columns):
         data = data.loc[:, not_null_cols]
 
+    # Setup for progress callback and make initial call
+    # Assume 1 unit for preprocessing, n for replace nans, n for make categorical and (n*n+n/2) for main calculation loop
+    n = len(data.columns)
+    total_loops = 1 + 2 * n + (n * n + n) / 2
```

Comment:
I think the comment is different from this; is it (n * n + n) / 2 or (n * n + n/2)? Where should the division by 2 go?

Reply:
@angela97lin Good catch. The code was correct, and the comment was wrong. I will fix.